controllers/apply/digital-funding-demo/form-model.js

```diff
@@ -0,0 +1,234 @@
+'use strict';
+const { check } = require('express-validator/check');
+const { PROJECT_LOCATIONS } = require('../reaching-communities/constants');
```

Comment:
Maybe copy-paste these into a new file too, just so we're not using any RC code (even if small)